src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double sideA;
+    private double sideB;
+    private double height;
+
+    public IsoscelesTrapezoid(double sideA, double sideB, double height, String color) {
+        super(color);
+        this.setSideA(sideA);
+        this.setSideB(sideB);
+        this.setHeight(height);
+    }
+
+    @Override
+    public double area() {
+        double s;
+        s = (((getSideA() + getSideB()) / 2) * getHeight());
```

Comment:
You can directly return the result of the expression without using an additional variable 's'.